README.md

```diff
@@ -90,6 +91,24 @@ To build the Deployment Manager, change to the `api` directory, which contains t
 * [Environment Endpoints API](#environment-endpoints-api)
   * [GET /environment/endpoints](#list-environment-variables-known-to-the-deployment-manager)
 
+
+## Base URL
+
+All API paths below are relative to a base URL is defined by schemes, host, port and base path on the root level of this API specification.
+
+```
+<scheme>://<host>:<port>/<base path>
+```
+
+By default, the API uses 'https' scheme as the transfer protocol. Host is the domain name or hostname that serves the API. In order to access the API outside PNDA security perimeter, it has to via knox service. Therefore the domain name or hostname has to be resolvable via public or private DNS service. If no domain name is provided when provisioning a PNDA cluster, the default domain is assigned by deault as `knox.service.dc1.pnda.local`. To access the deployment management API, the base path, `/gateway/pnda/deployment`, must be used as prefixes for all API paths. 
```

Comment:
This is no longer true - it's mandatory to supply the FQDN when creating the cluster